src/More/ExposureListDebugScreen.tsx

```diff
@@ -1,95 +1,41 @@
-import React, { useEffect, useState } from "react"
-import {
-  Alert,
-  BackHandler,
-  FlatList,
-  StyleSheet,
-  View,
-  Text,
-  NativeModules,
-} from "react-native"
+import React, { FunctionComponent } from "react"
+import { FlatList, StyleSheet, View, Text } from "react-native"
 
 import { GlobalText } from "../components/GlobalText"
-import { NavigationProp } from "../navigation"
 
 import dayjs from "dayjs"
-import { RawExposure } from "../gaen/dataConverters"
 
 import { Typography } from "../styles"
+import { useExposureContext } from "../ExposureContext"
 
-type ENLocalExposureScreenProp = {
-  navigation: NavigationProp
-}
-
-type DebugExposure = {
-  id: string
-  date: string
-}
-
-const ExposureListDebugScreen = ({
-  navigation,
-}: ENLocalExposureScreenProp): JSX.Element => {
-  const initialExposures: DebugExposure[] = []
-
-  const fetchExposures = async () => {
```

Comment:
Not sure if I'm missing something here, but is there a reason why we would want to fetch exposures independently in this screen when we are already using the same function to set current exposures in the `ExposureContext`? It seems like we already have the information

Reply:
Cool, yeah the function is already exposed through the context, so I can add that back into a useEffect hook. This would be easy to move to a button press in the future if we wanted!